### PR TITLE
Update fluent-bit to 3.1.8

### PR DESCRIPTION
**Pull request by reegnz**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 VERSION?=$(shell cat VERSION | tr -d " \t\n\r")
 # Image URL to use all building/pushing image targets
-FB_IMG ?= kubesphere/fluent-bit:v3.1.7
-FB_IMG_DEBUG ?= kubesphere/fluent-bit:v3.1.7-debug
+FB_IMG ?= ghcr.io/fluent/fluent-operator/fluent-bit:v3.1.8
+FB_IMG_DEBUG ?= ghcr.io/fluent/fluent-operator/fluent-bit:v3.1.8-debug
 FD_IMG ?= ghcr.io/fluent/fluent-operator/fluentd:v1.17.0
 FO_IMG ?= kubesphere/fluent-operator:$(VERSION)
 FD_IMG_BASE ?= ghcr.io/fluent/fluent-operator/fluentd:v1.17.0-arm64-base
```
Comment (wenchajun) [inline Makefile:7]: Is it possible to modify it here as well?

Comment (reegnz) [reply Makefile:7]: You mean the kubesphere stuff? I want to handle fluent-operator in a separate PR as I'm also handling fluent-bit separately: https://github.com/fluent/fluent-operator/pull/1357

**Review by benjaminhuo — APPROVED** [review]

**Comment by reegnz** [comment]
@benjaminhuo could you also trigger a build so there's a new 3.1.8 watcher image published?

**Comment by joshuabaird** [comment]
> @benjaminhuo could you also trigger a build so there's a new 3.1.8 watcher image published? AFAICT this is a manually triggered build: https://github.com/fluent/fluent-operator/blob/master/.github/workflows/build-fb-image.yaml#L4

@benjaminhuo @reegnz I'm building the v3.1.8 images now.

**Pull request merged**